superset/config.py

```diff
@@ -837,7 +837,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # Enables the replacement react views for all the FAB views (list, edit, show) with
 # designs introduced in SIP-34: https://github.com/apache/incubator-superset/issues/8976
 # This is a work in progress so not all features available in FAB have been implemented
-ENABLE_REACT_CRUD_VIEWS = False
+ENABLE_REACT_CRUD_VIEWS = os.environ.get("ENABLE_REACT_CRUD_VIEWS") or False
```

Comment:
This allows `ENABLE_REACT_CRUD_VIEWS` to be editable in e2e yaml file

Reply:
Should this be on the actual config.py or does it make more sense in the test config?